phylogenetic/README.md

```diff
@@ -0,0 +1,71 @@
+# nextstrain.org/dengue
+
+This is the [Nextstrain](https://nextstrain.org) build for dengue. Output from this build is visible at
+[nextstrain.org/dengue](https://nextstrain.org/dengue).
+
+
+## Software requirements
+
+Follow the [standard installation instructions](https://docs.nextstrain.org/en/latest/install.html) for Nextstrain's suite of software tools.
+
+## Usage
+
+If you're unfamiliar with Nextstrain builds, you may want to follow our
+[Running a Pathogen Workflow guide][] first and then come back here.
+
+The easiest way to run this pathogen build is using the Nextstrain
+command-line tool:
+
+    nextstrain build .
+
+Build output goes into the directories `data/`, `results/` and `auspice/`.
+
+Once you've run the build, you can view the results in auspice:
+
+    nextstrain view auspice/
+
+
+## Configuration
+
+Configuration takes place entirely with the `Snakefile`. This can be read top-to-bottom, each rule
+specifies its file inputs and output and also its parameters. There is little redirection and each
+rule should be able to be reasoned with on its own.
+
+
+### Using GenBank data
+
+This build starts by pulling preprocessed sequence and metadata files from: 
+
+* https://data.nextstrain.org/files/dengue/sequences_all.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_all.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv1.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv1.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv2.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv2.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv3.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv3.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv4.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv4.tsv.zst
+
+The above datasets have been preprocessed and cleaned from GenBank and are updated at regular intervals. 
+
+### Using example data
+
+Alternatively, you can run the build using the
+example data provided in this repository.  Before running the build, copy the
+example sequences into the `data/` directory like so:
+
+    mkdir -p data/
+    cp example_data/dengue* data/
```

Comment:
One other benefit to having the CI profile is users no longer have to manually copy over the example data.
These instructions can be updated to use
```
nextstrain build .  --configfile profiles/ci/profiles_config.yaml
```